modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/GoServerCodegen.java

```diff
@@ -185,6 +185,13 @@ public void processOpts() {
             additionalProperties.put("featureCORS", corsFeatureEnabled);
         }
 
+        if (additionalProperties.containsKey(CodegenConstants.ENUM_CLASS_PREFIX)) {
+            setEnumClassPrefix(Boolean.parseBoolean(additionalProperties.get(CodegenConstants.ENUM_CLASS_PREFIX).toString()));
+            if (enumClassPrefix) {
+                additionalProperties.put(CodegenConstants.ENUM_CLASS_PREFIX, "true");
```

Comment:
@mraineri Thanks for the PR. What about using `true` instead of `"true"`?

Reply:
It seemed like this pattern was used elsewhere, but I can make that change.

Reply:
I'll fix those with another PR. Thanks for pointing it out.

Reply:
Filed https://github.com/OpenAPITools/openapi-generator/pull/7134